jenkins/job-configs/kubernetes-jenkins/sen-lu-test-config-upload.yaml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: 'senlu-test-config-upload'
+    description: 'Build testgrid config from github and upload to gcs'
+    node: 'master'
+    properties:
+        - build-discarder:
+            num-to-keep: 20
+    triggers:
+        - pollscm:
+            cron: '* * * * *'
+    scm:
+        - git:
+            branches:
+            - config_external
+            browser: githubweb
+            browser-url: https://github.com/krzyzacy/test-infra
+            basedir: go/src/k8s.io/test-infra
+            skip-tag: true
+            url: https://github.com/krzyzacy/test-infra
+            wipe-workspace: false
+    wrappers:
+        - e2e-credentials-binding
+        - inject:
+            properties-content: |
+                GOROOT=/usr/local/go
+                GOPATH=$WORKSPACE/go
+                PATH=$PATH:$GOROOT/bin:$WORKSPACE/go/bin
+        - workspace-cleanup:
+            dirmatch: true
+            external-deletion-command: 'sudo rm -rf %s'
+    builders:
+        - activate-gce-service-account
+        - shell: |
+            export CONFIGDIR=k8s.io/test-infra/testgrid/config
+            go run $CONFIGDIR/main.go $CONFIGDIR/config.yaml $CONFIGDIR/config
```

Comment:
this might not work - the config file will be in `go/src/k8s.io/test-infra/testgrid/config`, but you're missing the `go/src` from the arguments here.